css/index.css

```diff
@@ -143,4 +290,129 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+.box1 {
+    background-color: teal;
+}
+
+/* About */
+
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Roboto', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-size: 18px;
+    margin-bottom: 15px;
+    font-family: 'Rubik', sans-serif;
+    font-weight: bolder;
+}
+
+p {
+    line-height: 1.4;
+    font-weight: bolder;
+}
+
+.containerabout-page {
+    width: 800px;
+    margin: 0 auto;
+}
+
+.mainheader {
+    display: flex;
+    justify-content: space-between;
+    margin: 2rem 0;
+}
+.mainheader a {
+    text-decoration: none;
+    color: black;
+    margin-top: 1rem;
+    margin-right: 1rem;
+    font-weight: bolder;
+}
+.jumbopicabout {
+    margin-bottom: 2rem;
+    margin-left: 22rem;
+}
+.top-content {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: left;
+    margin-bottom: 2rem;
+    padding: 0 1%;
+    padding-bottom: 2rem;
+    border-bottom: 1px dashed black;
+}
+* .textcontainer {
+    width: 50%;
+    padding-right: 80px;
+}
+* .textcontainer h3 {
+    margin-top: 1rem;
+}
+* .textcontainer p {
+    margin-bottom: 2rem;
+}
+
+* .textcontainer a {
+    text-decoration: none;
+    color: black;
+    border: 2px solid;
+    border-radius: 10px;
+    padding: 5px 15px;
+    font-weight: bolder;
+    border-style: solid;
+}
+.secondcontent {
+    display: flex;
+    justify-content: left;
+    margin-bottom: 2rem;
+    padding: 0 1%;
+    padding-bottom: 2rem;
+}
+.thirdcontent {
+    display: flex;
+    justify-content: left;
+    margin-bottom: 2rem;
+    padding: 0 1%;
+    padding-bottom: 2rem;
+}
+* hr{
+    margin-top: 1rem;
+    width: 99%;
+    border-bottom: 1px dashed black;
+}
+.bottomcontent {
+    margin-left: 1rem;
+    margin-bottom: 1rem;
+}
+
+footer {
+    width: 100%;
+    background: black;
+}
+
+footer nav {
+    width: 60%;  
+    /* display: flex; */
+    justify-content: space-between; 
+    /* justify-content: space-evenly; */
```

Comment:
In lines 403 and 405, you have commented-out code.  If you don't need it, delete it :) Keeping your codebase clean of unused code is a great habit to build early on.